src/Microsoft.SymbolManifestGenerator/Microsoft.SymbolManifestGenerator.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="System.Text.Json" Version="8.0.3" />
+      <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
Shouldn't we be using the versions.props property and bump that for the whole repo? @hoyosjs